packages/patternfly-4/react-core/src/components/Dropdown/DropdownItem.js

```diff
@@ -25,41 +28,65 @@ const defaultProps = {
   isHovered: false,
   component: 'a',
   isDisabled: false,
-  href: '#'
+  href: '#',
+  onSelect: Function.prototype,
```

Comment:
Missing from `propTypes` :)

Reply:
Oops, I'm wrong, we need to do the opposite.  I think we need to remove `onSelect` from `defaultProps`